plugin/api.go

```diff
@@ -125,6 +125,9 @@ type API interface {
 	// GetChannelByNameForTeamName gets a channel by its name, given a team name.
 	GetChannelByNameForTeamName(teamName, channelName string, includeDeleted bool) (*model.Channel, *model.AppError)
 
+	// GetChannelStats gets statistics for a channel.
```

Comment:
Can you add a comment indicating when this API became available?
```suggestion
	// GetChannelStats gets statistics for a channel.
	//
	// Minimum server version: 5.6
```